.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__/
+.ipynb_checkpoints/
```

Comment:
This change is irrelevant. It should move to your global gitignore.

Reply:
> Thanks, rebased an merged to the next branch.
Thanks